docs/design/autoware-interfaces/components/sensing.md

```diff
@@ -0,0 +1,52 @@
+# Sensing
+
+![Node diagram](images/Sensing-Bus-ODD-Architecture.drawio.svg)
+
+## Inputs
+
+### The sensing module contains the driver for the sensor, so there is no input
+
+## Output
+
+### Camera image
+
+Image data from camera. Used by the Perception.
+
+- [sensor_msgs/Image](https://github.com/ros2/common_interfaces/blob/rolling/sensor_msgs/msg/Image.msg)
+
+### Radar tracks
+
+Tracks from radar. Used by the Perception.
+
+- [radar_msgs/RadarTracks](https://docs.ros.org/en/noetic/api/radar_msgs/html/msg/RadarTracks.html)
+
+### Radar pointcloud
```

Comment:
From [my initial proposal](https://github.com/orgs/autowarefoundation/discussions/2531#discussion-4173490), we suggest

- [ros-perception/radar_msgs/msg/RadarScan.msg](https://github.com/ros-perception/radar_msgs/blob/ros2/msg/RadarScan.msg) for radar pointcloud.
- [ros-perception/radar_msgs/msg/RadarTracks.msg](https://github.com/ros-perception/radar_msgs/blob/ros2/msg/RadarTracks.msg) for radar object.

If you follow our proposal as [your comment](https://github.com/orgs/autowarefoundation/discussions/2531#discussioncomment-5977058), 
I think radar section of this document should be written as below.

```
### Radar objects

Objects information from radar. Used by the Perception.

- [radar_msgs/RadarTracks](https://docs.ros.org/en/noetic/api/radar_msgs/html/msg/RadarTracks.html)

### Radar pointcloud

Pointcloud from radar. Used by the Perception.

- [ros-perception/radar_msgs/msg/RadarScan.msg](https://github.com/ros-perception/radar_msgs/blob/ros2/msg/RadarScan.msg) for radar pointcloud.
```


If we need to continue to discuss radar data definition, 
I agree with [this comment](https://github.com/autowarefoundation/autoware-documentation/pull/328#discussion_r1190713838) and I would like to leave it TBD for now.

Reply:
@scepter914 ,

So we should differentiate between radar point clouds and radar objects, is that correct?

```md
### Radar objects

Objects information from radar. Used by the Perception.

- [radar_msgs/msg/RadarTracks.msg](https://github.com/ros-perception/radar_msgs/blob/ros2/msg/RadarTracks.msg)

### Radar pointcloud

Point cloud from radar. Used by the Perception.

- [radar_msgs/msg/RadarScan.msg](https://github.com/ros-perception/radar_msgs/blob/ros2/msg/RadarScan.msg) for radar pointcloud.
```

I've fixed the link for the radar track. I think we don't need to discuss it further for now and we can go with your proposal.

Reply:
From yesterday's discussion, we agreed that we can go with RadarTrack and RadarScan messages.

Reply:
Radar's description has been updated

Reply:
@xmfcx 

Thank you for your consideration.

> So we should differentiate between radar point clouds and radar objects, is that correct?

Yes.
The output specification depends on radar device.